### DISCUSSION TITLE
Copilot Access Ended Even Though Still a Student

### DISCUSSION THREAD

**Discussion by PressJump** [community/community#43624] [GitHub Education] [answered]
### Select Topic Area

Bug

### Body

Hey,
I had signed up for a student account in june after using copilot technical preview which had given me student access. At some point I got an email that my access was extended and that it would end today. I had seen this in another issue on this discussions board so I thought nothing of it as it was answered as a bug also. Today it ended like the email said with the message "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.". Even though I am on a student account that is still verified. 

With a verified student account and after methods listed for troubleshooting I checked my GitHub page for copilot in automation and yes, it was making me go ahead an purchase a plan. I am not too sure If I need to re-verify my student account but when tried to do so it did not let me so I am guessing thats not it.

**Comment by jongwooo** [comment]
@PressJump You can create related ticket on [GitHub Support](https://support.github.com/request) portal.

**Reply by PressJump** [reply]
Hey I have just created a ticket for this issue. It looks like alot of people were having this issue a month ago from the sources I have found before making this post and after. Today was the actual ending date for multiple cases I have seen going around. 2023-01-09.

**Reply by jongwooo** [reply]
@PressJump I'm also a student who uses Copilot, but I am still using it well. It seems to be a problem that occurs occasionally. I hope the problem is resolved well.

**Reply by Leandroglez39** [reply]
I have the same issue

**Reply by rakibur992** [reply]
I have the same problem. I am using Student Global Campus

**Reply by xiaodong2008** [reply]
I have the same problem.

**Reply by ayogun** [reply]
Same here

**Reply by shawntai** [reply]
Yo same

**Reply by jussihailu1** [reply]
jup

**Reply by omgitsmj24** [reply]
same

**Reply by duyonix** [reply]
same here

**Reply by codergautam** [reply]
me too :P

**Reply by asif256000** [reply]
Well, a year later I am having this exact same issue. How did you solve it?

**Comment by ZoukiLi** [comment]
I have the same issue as you. My student certification has not expired, but Copilot is no longer available

**Comment by anhdv47** [comment]
I have the same issue as you

**Comment by otkrickey** [comment]
same issue on 9/Jan/2023

**Comment by akib1689** [comment]
Hi, I have have the same issue as you do keep us posted. If any update occurs

**Comment by DaKup** [comment]
I'm glad I'm not the only one...

Hope this will be fixed soon.

**Comment by antikkorps** [comment]
Hi me too.

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by Iloveyousomuchben** [comment]
me neither T T

**Comment by otkrickey** [comment]
Solved: #43634

**Reply by E-wave112** [reply]
Same here @PressJump 
The issue started occurring on my end just a few hours ago

**Reply by otkrickey** [reply]
Read this

https://github.com/orgs/community/discussions/43634#discussioncomment-4629351

**Comment by anhdv47** [comment]
My problem has been solved 09/01/2023
![image](https://user-images.githubusercontent.com/101366253/211264731-8998b5bf-f8d8-46d1-97f2-2c102e044b04.png)

**Comment by akib1689** [comment]
Looks like this thing is solved now

**Comment by shawntai** [comment]
Still not working as of now

**Reply by Mahyar24** [reply]
Me too! :(

**Comment by Akash1134** [comment, marked as answer]
Hi there, 

Thank you everyone for taking the time to report and we apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging into your IDE plugin, and Copilot should work as expected.
If you run into any further issues, please reply back in detail with any useful screenshots.

**Reply by PressJump** [reply]
Hey @Akash1134 
Thanks for the update on the issue. I have marked your comment as an answer for anyone new coming in. I have also gotten an update email and it looks like my renewal has been sent forward by a month.

**Reply by cameron-chen** [reply]
Hi @Akash1134 

Thank you for letting us know the case. I am still facing the issue: 
![Screenshot 2023-01-12 at 11 55 54 AM](https://user-images.githubusercontent.com/60129542/211972427-c9461920-aaf9-429f-b5cc-e89262249a85.png)

My status: a verified student
No subscription/access to Copilot is shown on "billing and plans" panel

Can you please help with the issue?

--------------
[UPDATE] I have gained the access again. Thank you!

I guess Copilot team is solving this issue gradually.

**Reply by mattbolanos** [reply]
Hi @Akash1134 ! I am still experiencing difficulties, getting the same 403 missing token error as shown above by cameron-chen's comment, despite still being verified as a student on my account.

**Reply by y9c** [reply]
Some problem for me:

```
[INFO] [auth] [2023-01-12T04:05:31.210Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-12T04:05:31.212Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Reply by reydajp** [reply]
Same problem for me too

**Reply by richardhyy** [reply]
Same problem here. And last month, I got the following email:
```
Hi there,Thank you for renewing your free access to GitHub Copilot. We hope you enjoy using GitHub Copilot and participating in the developer community!Your access to GitHub Copilot will expire on 2023-01-09.
```

**Reply by UmbraCi** [reply]
same issue：
Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended
student identify is not overdue

**Reply by Bart-Roels** [reply]
Same problem here

**Reply by Prabhat-codes** [reply]
Hi. 
Still a student here, and I have Github Pro because of the student learner pack as well, but unable to get  access still. 
![image](https://user-images.githubusercontent.com/55730118/213196820-51400363-731b-4fe4-87cb-2a2eb8424619.png)
Please help me out.

**Reply by khiron** [reply]
I am an accredited student.  On the github copilot signup page I only get the option for commercial access.

![image](https://user-images.githubusercontent.com/1835007/213206031-8601fe45-6860-4fc6-9259-a8a30a314207.png)

Copilot was working last semester.  It is currently no longer associated with my account (despite the fact that my account remains accredited as a student account).

**Reply by prakharrathi25** [reply]
I am an accredited student. On the github copilot signup page I only get the option for commercial access.

<img width="374" alt="image" src="https://user-images.githubusercontent.com/38958532/213270923-2e4f4b6b-33c1-418b-a7ce-a995073a915a.png">

Copilot was working last semester. It is currently no longer associated with my account (despite the fact that my account remains accredited as a student account).

**Reply by khiron** [reply]
Thank you.  It appears to be working again.

![image](https://user-images.githubusercontent.com/1835007/213432240-5547acde-80ec-4b25-a93a-a98ea6adf7e4.png)

**Reply by prakharrathi25** [reply]
Working now, thanks!

**Reply by rielmvp** [reply]
<img width="792" alt="Screen Shot 2023-02-01 at 10 12 26 PM" src="https://user-images.githubusercontent.com/83220394/216052905-34d5a552-058b-4767-9736-28f911077cfc.png">

Copilot was working but it seems that it is no longer linked to my account even though I use student email and my account seems to be accredited as a student account. Github username:rielmvp.

**Reply by danvufs** [reply]
> Hi there,
> 
> Thank you everyone for taking the time to report and we apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging into your IDE plugin, and Copilot should work as expected. If you run into any further issues, please reply back in detail with any useful screenshots.

Until now, my account is still: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended". Please double-check!

**Reply by khiron** [reply]
My student CoPilot access reverted back again to "thank you for partiticpating" and it's not available with pycharm or VS code.

**Reply by Pedro08994** [reply]
> Gracias. Parece estar funcionando de nuevo.
> 
> ![imagen](https://user-images.githubusercontent.com/1835007/213432240-5547acde-80ec-4b25-a93a-a98ea6adf7e4.png)

**Comment by codergautam** [comment]
<img width="526" alt="image" src="https://user-images.githubusercontent.com/66850133/211970356-760aabc7-818c-494b-b007-adf07bab22c2.png">

I am glad this is getting solved for people, but recently I started having this issue as well.  Should I create a new support ticket? Thanks

**Reply by ninh-nd** [reply]
Guess they haven't really fixed it completely. Same boat here

**Comment by davidlechuga** [comment]
i have the answe, just change the version of the plugin and reload the window of vscode.
![7](https://user-images.githubusercontent.com/47370612/227614857-5aa2752d-59d1-48c3-9ac4-a7e27bb0e740.png)

**Comment by donRehan** [comment]
I am still getting this error , I have a student account.
I tried logging out and logging in , Uninstalling Copilot and installing it and nothing seems to change.

Invalid copilot token: missing token: 403 
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."

**Comment by shahadat3031diu** [comment]
Student account is not working

**Comment by jacob-02** [comment]
I'm also facing the same issue
![image](https://github.com/community/community/assets/74947682/25653129-036f-4637-8bd0-c658ceefc3be)

**Comment by saltedfish68** [comment]
Hi there,
I have same issue!
could you pleas help me check as well? Thanks in advance!

**Reply by asif256000** [reply]
You can check if your account was flagged or not. Once I started getting these issues more than 3 weeks back, I poked around and saw that my account was flagged for some reason. Since then I have raised a ticket (3 weeks back), but am yet to get any response from the GitHub support team.

It is just sad that all my work is tied to this one account, which I can't use for so long because GitHub support is not taking any action for this long. In case your account is also flagged, you can search 'GitHub Reinstatement Request' and you will find where you have to initiate a ticket.

**Comment by goldmike1234** [comment]
Hi, I seem to have a similar issue that I can't seem to renew because it says I don't need to renew it again but I have no access to it "Error
Hi, michael! You were last verified as a student on . It is not necessary for you to re verify at this time. "

"Hi, goldmike1234! You were last verified as a student on Oct 17, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

I think it might be a github problem itself, I tried redoing everything even uninstalling the extension etc but it didn't work for me. I guess the bug its still out there
